client/lib/plans/plans-list.js

```diff
@@ -357,6 +369,98 @@ const getPlanBusinessDetails = () => ( {
 	getInferiorHiddenFeatures: () => [],
 } );
 
+const getPlanJetpackSecurityDailyDetails = () => ( {
+	group: constants.GROUP_JETPACK,
+	type: constants.TYPE_SECURITY,
+	getTitle: () =>
+		translate( 'Jetpack Security {{em}}Daily{{/em}}', { components: { em: <em /> } } ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'Jetpack Backup {{em}}Daily{{/em}}' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/get-new-product-data)